### DISCUSSION TITLE
custom name for the `other-links` section

### DISCUSSION THREAD

**Discussion by aghaynes** [quarto-dev/quarto-cli#11662] [Q&A] [answered]
### Description

When using the `other-links` feature, is it possible to use a custom piece of text rather than "Other links"? The same question might be relevant for the `format-links` and `code-links` sections perhaps... In the following example I might want it to say "Email services", for example.

```` qmd
---
title: "Untitled"
editor: visual
format: html
other-links:
  - text: gmail
    href: https://mail.google.com
    icon: envelope
    target: _blank
  - text: hotmail
    href: https://www.hotmail.com
    icon: envelope
    target: _blank
---
```` 

![image](https://github.com/user-attachments/assets/4487b8a8-1993-41ac-8ae2-1d9ed2c51738)

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/authoring/language.html> to set alternate languages or translations.

**Comment by aghaynes** [comment]
Perfect! Thanks! Maybe it's worth adding a reference to this on https://quarto.org/docs/output-formats/html-basics.html#code-links-and-other-links... It would have never have occured to me that such a thing might be a language option...

**Reply by mcanouil** [reply]
Changing any default text anywhere is a language/translation option.
So it would to add this everywhere.
